src/pages/ContentScripts/features/community-openrank-network/index.scss

```diff
@@ -0,0 +1,31 @@
+#details_table {
+  width: 95%;
+  margin: 10px;
+  tr:nth-child(even) {
+    background-color: #d6eeee;
+  }
+  th,
+  td {
+    border: 1px solid black;
+    text-align: center; /* 水平居中 */
+    vertical-align: middle; /* 垂直居中 */
```

Comment:
It is best to replace the Chinese comments in the code with English. And only keep necessary comments. Similar to some code that has been commented out, it can be removed.